Updated versions

kumadee opened the pull request.

Commit:
Updated versions

PirxDanford commented:
The topfloor/composer-cleanup-vcs-dirs is only meant to be used internally in the CI and should not be there. We need to remove it.

Commit:
Updated composer.json
- Removed php 7.4 specific dependencies
- Removed composer-cleanup-vcs as per the review comments

Commit:
updated versions after latest run

Commit:
flow theme version update for https://bugs.oxid-esales.com/view.php?id=7163

Commit:
Updates for composer v2 compatibility with all supported PHP
- fix for issue https://oxid-esales.atlassian.net/browse/OXDEV-4173
